### PR TITLE
bpo-37256: Wording in Request class docs

### PR DESCRIPTION
**Pull request by nsiregar**
change wording in `Request` class documentation

https://bugs.python.org/issue37256

### CODE REVIEW

**Review by mangrisano — APPROVED** [review]

File: Misc/NEWS.d/next/Documentation/2019-07-16-14-48-12.bpo-37256.qJTrBb.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+change wording in Request class
```
Comment (aeros) [inline Misc/NEWS.d/next/Documentation/2019-07-16-14-48-12.bpo-37256.qJTrBb.rst]: Specified location of change and added RST class and module tags ([devguide](https://devguide.python.org/documenting/#inline-markup)):
```suggestion
Fix wording of arguments for :class:`Request` in :mod:`urllib.request`
```

**Review by aeros — APPROVED** [review]
@nsiregar Welcome and thanks for the contribution!

I definitely agree with the suggested changes. Since there are three additional arguments, it should be "next" instead of "last". 

However, I would recommend making a small adjustment to the phrasing of the news entry to specify the module of the class, and use reST (markup language used by python docs):

**Review by CuriousLearner — APPROVED** [review]
Wow, nice catch! 💯 

LGTM 🌮 

Thank you for your contribution @nsiregar

**Comment by CuriousLearner** [comment]
Hiya @Mariatta @orsenthil 

Can you please have a look at this one?

Thanks!

**Review by orsenthil — APPROVED** [review]
Thank you, Sanyam.

**Pull request merged**